templates/cisco_nxos_show_ip_interface_brief.template

```diff
@@ -0,0 +1,8 @@
+Value Required INTF (\S+)
```

Comment:
Possibly try and merge master back into this branch, but possibly pushing up another commit might remove the cisco_nxos_show_ip_interface_brief changes since they're now in master.

Reply:
I think I got things squared away.  You should have 3 files coming up in this commit, not 4.  Sorry about that, I'm a bit new using github.  I'll try to check my work better in the future.

Let me know if you still see an issue and I can delete and recreate or get someone to help on this end.